.github/workflows/functions-firebase.yaml

```diff
@@ -36,12 +48,10 @@ jobs:
       with:
         node-version: 14
     - run: npm install
-      working-directory: functions/firebase
+      working-directory: ${{ matrix.package }}
     - run: npm test
-      working-directory: functions/firebase
+      working-directory: ${{ matrix.package }}
       env:
-        MOCHA_REPORTER_SUITENAME: functions_firebase
```

Comment:
I think these are needed for flakybot

Reply:
I could not find any evidence these were used outside of Kokoro. (https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/.kokoro/.mocharc.js)

Is there a way to see if the http sample I already moved is properly reporting to flakybot?

Reply:
I found a way to add these options to the GHA-invoked test run, without eliminating the output for a manual `npm test`. wdyt?